website_sale_secondary_unit/tests/test_website_sale_secondary_unit.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests.common import HttpCase
+
+
+class WebsiteSaleHttpCase(HttpCase):
+    def setUp(self):
+        super(WebsiteSaleHttpCase, self).setUp()
+        with self.cursor() as cr:
+            env = self.env(cr)
+            self.product_ipod = env.ref('product.product_product_11')
+            self.product_ipod.write({
+                'secondary_uom_ids': [
+                    (0, 0, {
+                        'name': 'box',
+                        'uom_id': self.product_ipod.uom_id.id,
+                        'factor': 10,
+                    }),
+                    (0, 0, {
+                        'name': 'box',
+                        'uom_id': self.product_ipod.uom_id.id,
+                        'factor': 20,
+                    }),
+                ],
+                'allow_uom_sell': True,
+                'website_published': True,
+            })
+
+    def test_ui_website(self):
+        """Test frontend tour."""
+        tour = (
+            "odoo.__DEBUG__.services['web_tour.tour']",
+            "website_sale_secondary_unit.tour",
```

Comment:
You have to search the tour by [the name used to register it](https://github.com/OCA/e-commerce/pull/275/files#diff-33dc1ed7fbbae44004358753bd9f7874R50), not by the JS module name.

Change this for: `"website_sale_secondary_unit"`.